src/components/AuditLogsTable/AuditLogsTableActions/AuditLogsTableActions.test.tsx

```diff
@@ -71,7 +71,7 @@ describe("AuditLogsTableActions", () => {
     await userEvent.click(screen.getByRole("button", { name: "Next page" }));
     expect(window.location.search).toEqual("?page=2");
     await userEvent.click(screen.getByRole("button", { name: "Refresh" }));
-    expect(window.location.search).toEqual("?page=1");
+    expect(window.location.search).toEqual("");
```

Comment:
In a number of places I've changed this to remove the page param so that it uses the default provided from the hook rather than explicitly setting it back to page 1.